app/dashboard/router.py

```diff
@@ -71,6 +73,21 @@ class BountySerializer(serializers.HyperlinkedModelSerializer):
     activities = ActivitySerializer(many=True)
     bounty_owner_email = serializers.SerializerMethodField('override_bounty_owner_email')
     bounty_owner_name = serializers.SerializerMethodField('override_bounty_owner_name')
+    resurfaced = serializers.SerializerMethodField('set_resurfaced_issue')
+
+    # check for extended issues and resurface them
+    def set_resurfaced_issue(self, issue):
+        issue_activities = issue.activities.all()
```

Comment:
Can we accomplish the same with a query?
Like 
```
issue_extended = issue.idx_status == 'open'
    and not issue.past_expiration_date
    and issue.activities.all().filter(activity_type='extend_expiration').exists()
```

^ This can remove the whole loop

Reply:
@pinkiebell awesome thanks implementing it like this